samples/matter/light_bulb/sample.yaml

```diff
@@ -47,3 +47,9 @@ tests:
     integration_platforms:
       - nrf5340dk_nrf5340_cpuapp
     platform_allow: nrf5340dk_nrf5340_cpuapp
+  # Sample to execute load tests
+  sample.matter.light_bulb.persistence_subscription:
```

Comment:
You have to select integration platforms, if you don't want it to run in integration builds you have to use quarantine configuration file. Maybe enable one of them for 7002 and another for 52840. I think the tag is redundant for now but it is a good idea to comment configurations if they are specific in testing context.